bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh

```diff
@@ -338,12 +338,34 @@ fi
 
 #Set Docker token using the BEARER token captured earlier
 if [ "${SENSOR_TYPE}" = "kpagent" ]; then
-    docker_api_token=$(curl_command "$cs_falcon_oauth_token" "https://$(cs_cloud)/kubernetes-protection/entities/integration/agent/v1?cluster_name=clustername&is_self_managed_cluster=true" | awk '/dockerAPIToken:/ {print $2}')
+    raw_docker_api_token=$(curl_command "$cs_falcon_oauth_token" "https://$(cs_cloud)/kubernetes-protection/entities/integration/agent/v1?cluster_name=clustername&is_self_managed_cluster=true")
+    docker_api_token=$(echo "$raw_docker_api_token" | awk '/dockerAPIToken:/ {print $2}')
 else
-    docker_api_token=$(curl_command "$cs_falcon_oauth_token" "https://$(cs_cloud)/container-security/entities/image-registry-credentials/v1" | json_value "token")
+    raw_docker_api_token=$(curl_command "$cs_falcon_oauth_token" "https://$(cs_cloud)/container-security/entities/image-registry-credentials/v1" )
+    docker_api_token=$(echo "$raw_docker_api_token" | json_value "token")
 fi
 ART_PASSWORD=$(echo "$docker_api_token" | sed 's/ *$//g' | sed 's/^ *//g')
 
+if [ -z "$ART_PASSWORD" ] ; then
+    echo "===================="
+    echo "API Cloud Response:"
+    echo "$raw_docker_api_token"
+    echo "===================="
+    echo "ART_PASSWORD is NULL, Please check:"
+    echo "1 - Your credentials are valid"
+    echo "2 - Your variables are not NULL"
+    echo "3 - You are have using the correct API Scopes (Falcon Image Download [read], Sensor Download [read], Kubernetes Protection [read])"
+    echo "4 - You have Cloud Security enabled in your tenant"
+    die "Can't get the CrowdStrike registry password"
+fi
```

Comment:
I would probably stray from using all this in the error output.. I would do something like this:
```suggestion
if [ -z "$ART_PASSWORD" ] ; then
    error_message="Failed to retrieve the CrowdStrike registry password. Response from API:
$raw_docker_api_token"

    die "$error_message"
fi
```

If they need additional messaging they should consult the README. If we are missing something in the README, then I would add it to this PR. Otherwise, keep it simple. Those are my 2 cents.

Reply:
That's not what I'm seeing and usually they contact support or SE for guidance. Here there is an issue we have the raw output if needed.
Steps 1 to 3 are the most common ones taking lot of time on our side.